docs/measurements/index.md

```diff
@@ -85,23 +95,59 @@ from qadence_protocols.measurements.utils_shadow import number_of_samples
 
 shadow_options = {"accuracy": 0.1, "confidence": 0.1}
 N, K = number_of_samples(observable, **shadow_options)
-shadow_measurement = Measurements(protocol=Measurements.SHADOW, options=shadow_options)
+shadow_measurement = Measurements(protocol=MeasurementProtocols.SHADOW, options=shadow_options)
 
 # Run the shadow experiment.
-estimated_values_shadow = shadow_measurement(model)
+estimated_values_shadow = shadow_measurement(model, return_expectations=True)
 
 print(f"Estimated expectation value shadow = {estimated_values_shadow}") # markdown-exec: hide
 ```
 
+## Getting shadows
+
+If we are interested in accessing the measurement data from shadows, we can access the measurement data via the `manager` attribute or simply pass `return_expectations=False` (a new round of measurements is performed and returned directly instead of computing expectation values) as follows:
+
+```python exec="on" source="material-block" session="measurements" result="json"
+
+measurements_shadows = shadow_measurement.manager.data
```

Comment:
Why not turning `MeasurementData` into a dataclass that holds `unitaries` and `measurements` as fields ? It'd be simpler than a key/value access imho.

Reply:
Also: would it make sense to make the manager transparent ? Ie. make user exposed fields directly accessible in `Measurements` ?